src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -25,16 +25,9 @@
 
 package javax.crypto;
 
-import java.security.AlgorithmParameters;
-import java.security.Key;
-import java.security.SecureRandom;
-import java.security.NoSuchAlgorithmException;
-import java.security.InvalidKeyException;
-import java.security.InvalidAlgorithmParameterException;
-import java.security.ProviderException;
-import java.security.spec.AlgorithmParameterSpec;
-
 import java.nio.ByteBuffer;
+import java.security.*;
```

Comment:
This is another one that some people will complain about.  I personally prefer this style, others prefer everything written out as long as it's not "too many."

Reply:
Keeping the change for now.